package/network/services/ppp/patches/201-mppe_mppc_1.1.patch

```diff
@@ -1,1518 +0,0 @@
-pppd: add support for MPPE and MPPC encryption and compression protocols
-
-This is a forward ported version of ppp-2.4.3-mppe-mppc-1.1.patch.gz found on
-http://mppe-mppc.alphacron.de/ .
-
-Signed-off-by: Jo-Philipp Wich <jo@mein.io>
```

Comment:
The MPPC functionality is not supported in upstream ppp, MPPE is supported in upstream and can be activated using `enable_microsoft_extensions` like it is done in this PR. 

Do we need MPPC support? I think it is ok to remove it.